### DISCUSSION TITLE
Re-clustering issues satellite_groups

### DISCUSSION THREAD

**Discussion by paulasoler** [stuart-lab/signac#618] [Q&A] [answered]
Hello! Thanks for your amazing work developing Signac. 
I have an issue when I apply a re-clustering of a specific cell type from the dataset. I have used the code that I attach below:

seurat <- seurat %>%
RunTFIDF() %>%
FindTopFeatures(min.cutoff = "q0") %>%
RunSVD()
RunHarmony(dims = 2:40, group.by.vars = "gem_id", reduction = "lsi") %>%
RunUMAP(reduction = "harmony", dims = 2:40) %>%
FindNeighbors(reduction = "harmony", dims =2:40)

I obtained a main cluster that determines the cell of interest and around it a kind of satellite groups and it's something that is recursive, meaning if I delete the satellites more satellites appear.  Have you ever seen this before ? I am doing a stringent analysis on the poor quality cells/doublets....

![](https://user-images.githubusercontent.com/3385292/116682909-2b8d8d00-a9af-11eb-9d32-1b2d1d9a93ac.png)

Thanks :)

**Comment by timoast** [comment, marked as answer]
This looks weird, and is not something I've seen before. Can you try this without the Harmony step, to see if it's related to that, and also try setting `min.cutoff="q5"` in `FindTopFeatures()`? Could you also show the code you used to process the object before this step?

**Reply by paulasoler** [reply]
As you recommended, I set the min.cutoff="q5". I am attaching you the resulting plot before and after Harmony correction and it looks like that it's something related to that. However, I need to correct for the differences between the applied techniques. Any suggestions?

![Captura de pantalla 2021-05-20 a las 15 25 52 (2)](https://user-images.githubusercontent.com/3385292/118987025-2bfebf80-b980-11eb-85ae-8e2abd39d365.png)

**Reply by timoast** [reply]
Looks like setting `min.cutoff='q5'` has solved it, not sure what you mean that you need to correct for differences between the applied techniques....isn't that the goal of the harmony step?

**Reply by timoast** [reply]
You can also take a look at the signac integration vignette: https://satijalab.org/signac/articles/integrate_atac.html
